#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
float64 max_offset
float64 lateral_gain
float64 heading_gain
float64 speed
float64 low_speed
float64 full_speed_curvature
float64 low_speed_curvature
float64 deceleration
bool use_curvature_control